device-types/Cisco/C9800-L-C.yaml

```diff
@@ -17,19 +21,19 @@ interfaces:
   - name: Redundancy Port
     type: 1000base-t
     mgmt_only: true
-  - name: TenGigabitEthernet1
-    type: 10gbase-t
-  - name: TenGigabitEthernet2
-    type: 10gbase-t
-  - name: GigabitEthernet1
+  - name: TwoGigabitEthernet0/0/0
     type: 2.5gbase-t
-  - name: GigabitEthernet2
+  - name: TwoGigabitEthernet0/0/1
```

Comment:
Is this actually the name of the interface from "show interface status"?

Reply:
In our deployment it is. This is due to it being a 2.5 Gigabit Interface which is referenced to cisco as TwoGigabitEthernet... the `/` placement I'm not 100% on as we run ours in cluster.. but their cluster is active/passive synced and it doesn't address the interfaces separately like you might see in a switchstack so I do think its supposed to be the full `0/x/x` 

And yes oddly they number these units starting at 0... possibly hold over from aireos days I think.

Reply:
Are you able to show us an output from the cli? If so that would be great

Also, is this by default or does it default to the removed name? We want to keep this repo with the defaults as much as possible

Reply:
Hey @danner26 - I was looking to see if easy way to verify whether interface name changes after joining a cluster or not. Was able to reference [this youtube video](https://youtu.be/UhZedbZ1pIA?si=oBn2d__QrMSeH-JG&t=770) where they setup a stand alone non-ha 9800-L to confirm yes the names are still 0/x/x regardless of cluster status -- this also shows the interface names as `Tw...` as discussed

If a cli output is still needed I can get one will just need to redact a bunch from it --